### PR TITLE
Jetty 12.0.x fix cache issue with maven invoker

### PR DESCRIPTION
**Pull request by olamy**

### CODE REVIEW

File: .mvn/maven-build-cache-config.xml [hunk 1]
```diff
@@ -94,12 +99,5 @@
       </plugins>
     </reconcile>
   </executionControl>
-  <output>
```
Comment (olamy) [inline .mvn/maven-build-cache-config.xml:97]: yup we need jetty-home.zip cached arghh

File: jetty-ee10/jetty-ee10-webapp/src/test/java/org/eclipse/jetty/ee10/webapp/MetaInfConfigurationTest.java [hunk 2]
```diff
@@ -49,17 +49,6 @@
 @ExtendWith(WorkDirExtension.class)
 public class MetaInfConfigurationTest
 {
-    @BeforeEach
```
Comment (olamy) [inline jetty-ee10/jetty-ee10-webapp/src/test/java/org/eclipse/jetty/ee10/webapp/MetaInfConfigurationTest.java:52]: this cannot be done with parrallel tests or can be flaky

File: jetty-ee10/jetty-ee10-webapp/src/test/java/org/eclipse/jetty/ee10/webapp/MetaInfConfigurationTest.java [hunk 3]
```diff
@@ -49,17 +49,6 @@
...
-    public void beforeEach()
-    {
-        assertThat(FileSystemPool.INSTANCE.mounts(), empty());
-    }
-
-    @AfterEach
-    public void tearDown()
-    {
-        assertThat(FileSystemPool.INSTANCE.mounts(), empty());
```
Comment (joakime) [inline jetty-ee10/jetty-ee10-webapp/src/test/java/org/eclipse/jetty/ee10/webapp/MetaInfConfigurationTest.java]: Then all of these tests shouldn't be parallelized.
This is an important step in our testing, don't remove it.
Perhaps these tests should be forked instead?

**Review by joakime — DISMISSED** [review]
The removal of the FileSystemPool assertions bothers me

**Review by gregw — APPROVED** [review]

**Pull request merged**